ui/webui/resources/br_elements/br_toolbar/br_toolbar.html

```diff
@@ -38,21 +38,20 @@
 
   .nav-item {
     align-items: center;
-    color: #fff;
+    color: var(--leo-color-white);
     cursor: pointer;
     display: flex;
     /* update br_toolbar.js font-load detection
     if font-weight or name changes */
-    font-family: Poppins;
     font-size: 16px;
-    font-weight: 300;
-    letter-spacing: 0.6px;
-    /* hide until fonts are loaded */
+    font-weight: 500;
```

Comment:
We don't seem to have a `leo` variable with this weight - should this maybe be one of
1. --leo-font-large-link: 400 16px/24px Inter Variable;
2.  --leo-font-large-regular: 400 16px/24px Inter Variable;

```suggestion
    font: var(--leo-font-the-right-one);
```

Reply:
Awesome, makes sense